__tests__/integration/configuration/privnet.yml

```diff
@@ -0,0 +1,32 @@
+# This file is the Private Network Configuration for the Fullnode
+# It is consumed by the docker-compose.yml file on the integration folder.
+# For more information, refer to:
+# https://github.com/HathorNetwork/rfcs/blob/master/text/0033-private-network-guide.md
+
+# This genesis adds the funds to the following wallet seed:
+# avocado spot town typical traffic vault danger century property shallow divorce festival spend attack anchor afford rotate green audit adjust fade wagon depart level
+
+P2PKH_VERSION_BYTE: x49
+MULTISIG_VERSION_BYTE: x87
```

Comment:
```suggestion
P2PKH_VERSION_BYTE: 0x49
MULTISIG_VERSION_BYTE: 0x87
```

Reply:
I'm just following what we have in the full node: https://github.com/HathorNetwork/hathor-core/blob/master/hathor/conf/testnet.yml#L1